mscore/keyb.cpp

```diff
@@ -213,10 +213,10 @@ void ScoreView::editKey(QKeyEvent* ev)
             }
 
 
-#ifdef Q_OS_WIN // Japenese IME on Windows needs to know when Contrl/Alt/Shift/CapsLock is pressed while in predit
+#ifdef Q_OS_WIN // ignore Contrl/Alt/Shift/CapsLock while in predit
```

Comment:
I was trying the chinese input on (parabola) arch linux Mate Desktop.  I wasn't able to get the fcitx input method framework working in either QtCreator or MuseScore.  But I did try the ibus framework, and was able to input chinese pinyin into MuseScore 2.0.3 (arch linux package) and self-compiled master and also easily switch with Super-SpaceBar.  But interestingly I couldn't input chinese with the App Images (both 2.1 and master)...but I suspect that issue is more involved and not worth solving, since can always wait for their distro's package to be updated.

So I guess that only leaves mac to test, but I don't have a mac.